### ISSUE TITLE
(Windows) Files import with "\" treated as a character in file name rather than path separator

### ISSUE THREAD

**Issue by epage** [LucioFranco/ghp-rs#8]
Cobalt has calls `import_dir` to import a statically generated site.  @natejms has reported that on Windows (using msys?) that "\" is used in the paths but is treated as a file name character rather than as a path separator.

See https://github.com/natejms/cobalt-org.github.io/tree/master

OS: Windows 10

My best guesses at the reproduction steps:
1. Install cobalt
2. `cobalt init`
3. `cobalt import`

**Comment by LucioFranco** [comment]
Hmm. I took a look and it is adding a `\` before the filename for me. I think this issue arises due to it actually running on windows but in a fake linux env. I will need to take some more time to fix this but will probably need to filter out any extra `\`.

I wonder if there are any open issues with the path crate.

**Comment by epage** [comment]
Wonder if this is related: https://github.com/sharkdp/fd/issues/153

**Comment by jedahan** [comment]
Possibly also jedahan/zr#14

**Comment by LucioFranco** [comment]
Yeah this is def related but I don’t like that possible fix. Will have to wait and see how they finally decide to fix it.
